lib/plugins/aws/deploy/compile/events/apiGateway/lib/methods.js

```diff
@@ -1,112 +1,114 @@
 'use strict';
 
 const BbPromise = require('bluebird');
-const forEach = require('lodash').forEach;
-const merge = require('lodash').merge;
+const _ = require('lodash');
```

Comment:
Yes. Lodash is a kind of rewrite (faster) of underscore.js and underscore uses `_` as a symbol (like `$` for jQuery.